skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -15,33 +16,32 @@
 sys.path.insert(0, os.path.abspath(path))
 
 # Imports
-import pytest
+from time import sleep
 from mock import MagicMock
+from PyTango import DevFailed, DevState
+from devicetest import DeviceTestCase, main
+from SKALogger import SKALogger
 
-from PyTango import DevState
-
-# PROTECTED REGION ID(SKALogger.test_additional_imports) ENABLED START #
-# PROTECTED REGION END #    //  SKALogger.test_additional_imports
+# Note:
+#
+# Since the device uses an inner thread, it is necessary to
+# wait during the tests in order the let the device update itself.
+# Hence, the sleep calls have to be secured enough not to produce
+# any inconsistent behavior. However, the unittests need to run fast.
+# Here, we use a factor 3 between the read period and the sleep calls.
+#
+# Look at devicetest examples for more advanced testing
 
 
 # Device test case
-@pytest.mark.usefixtures("tango_context", "initialize_device")
-# PROTECTED REGION ID(SKALogger.test_SKALogger_decorators) ENABLED START #
-# PROTECTED REGION END #    //  SKALogger.test_SKALogger_decorators
-class TestSKALogger(object):
+class SKALoggerDeviceTestCase(DeviceTestCase):
     """Test case for packet generation."""
-
-    properties = {
-        'SkaLevel': '4',
-        'MetricList': 'healthState',
-        'GroupDefinitions': '',
-        'CentralLoggingTarget': '',
-        'ElementLoggingTarget': '',
-        'StorageLoggingTarget': 'localhost',
-        'CentralLoggingLevelDefault': '2',
-        'ElementLoggingLevelDefault': '3',
-        'StorageLoggingLevelDefault': '4',
-        }
+    # PROTECTED REGION ID(SKALogger.test_additionnal_import) ENABLED START #
+    # PROTECTED REGION END #    //  SKALogger.test_additionnal_import
+    device = SKALogger
+    properties = {'SkaLevel': '4', 'MetricList': 'healthState', 'GroupDefinitions': '', 'CentralLoggingTarget': '', 'ElementLoggingTarget': '', 'StorageLoggingTarget': 'localhost', 'CentralLoggingLevelDefault': '2', 'ElementLoggingLevelDefault': '3', 'StorageLoggingLevelDefault': '4', 
```

Comment:
Keep the line length up to 90 chars as per coding standards.

Reply:
Done.

Reply:
Done